.travis.yml

```diff
@@ -1,9 +1,15 @@
 dist: trusty
+sudo: required
+before_install:
+  - source .travis/before_install_$TRAVIS_OS_NAME.sh
```

Comment:
Please use the following syntax:

```yml
addons:
  apt:
    packages:
    - poppler-utils
```

We should not be changing the sudo setting.